packages/tendermint-rpc/src/rpcclients/httpclient.ts

```diff
@@ -65,29 +65,67 @@ export interface HttpEndpoint {
   readonly headers: Record<string, string>;
 }
 
+export interface HttpClientOptions {
+  dispatchInterval: number;
+}
+
+export const defaultHttpClientOptions: HttpClientOptions = { dispatchInterval: 0 };
+
 export class HttpClient implements RpcClient {
```

Comment:
Let's make this a second client please (which we then can use by default once tested and stabilized). This keeps us flexible testing both implementations and going back if we run into trouble for some cases.